resources/[gameplay]/gcshop/horns/horns_c.lua

```diff
@@ -711,7 +711,7 @@ function createSoundForCar(car, horn)
 			guiSetAlpha(icon[car], globalalpha)
 			guiSetSize(icon[car], relx, rely, false)
 			if(screenX and screenY) then
-				guiSetPosition(icon[car], screenX, screenY, false)
+				guiSetPosition(icon[car], screenX-50, screenY+50, false)
```

Comment:
probably need to move a bit more, sadly I can't experiment on this at the moment as I have no local server etc